spec/lib/pdf_filler/nj1040_pdf_spec.rb

```diff
@@ -92,6 +92,71 @@
       end
     end
 
+    describe "exemptions" do
+      context "single filer" do
+        let(:submission) {
+          create :efile_submission, tax_return: nil, data_source: create(
+            :state_file_nj_intake,
+            primary_first_name: "Bert",
+            primary_last_name: "Muppet",
+            primary_middle_initial: "S",
```

Comment:
[pebble] these name lines don't seem relevant to what's being tested, can they be deleted? Here and in the tests below as well.